src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -54,15 +54,18 @@ public void Initalize(IConfiguration configuration)
         {
             if (Initialized) return;
 
+            _logger.LogInformation($"Detecting CSX files in '{_env.TargetDirectory}'.");
+
+            // Nothing to do if there are no CSX files
+            var allCsxFiles = _fileSystemHelper.GetFiles("**/*.csx").ToArray();
+
             _scriptOptions = new ScriptOptions();
+            _scriptOptions.CsxFiles = allCsxFiles;
```

Comment:
actually it would be better to just add an extra argument to `CreateScriptContext(...)`.
`ScriptOptions` is bound from the `omnisharp.json` so it's not good idea to use it for just transporting objects or data from one place to another